src/cluster/cluster.cc

```diff
@@ -37,11 +37,11 @@
 #include "time_util.h"
 
 ClusterNode::ClusterNode(std::string id, std::string host, int port, int role, std::string master_id,
-                         std::bitset<kClusterSlots> slots)
+                         std::bitset<kClusterSlots> const &slots)
     : id(std::move(id)), host(std::move(host)), port(port), role(role), master_id(std::move(master_id)), slots(slots) {}
```

Comment:
what about using std::move?

Reply:
I don't think it will work. The data in bitset is on the stack.